pom.xml

```diff
@@ -83,6 +83,21 @@
 			<artifactId>spring-security-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.sendinblue</groupId>
+			<artifactId>sib-api-v3-sdk</artifactId>
+			<version>3.0.0</version>
+		</dependency>
+		<dependency>
+			<groupId>xerces</groupId>
+			<artifactId>xercesImpl</artifactId>
+			<version>2.9.1</version>
+		</dependency>
+		<dependency>
```

Comment:
please remove that, it is already imported. All you need to do is refresh your maven view in Idea.
When you are not sure, you can always build from terminal using
`mvn clean install`. If it works - the problem is in IDEA